commit 401e975e5950ad9f64f6348c21f6c05e65e0de91
Author: Marcel Ziswiler <marcel.ziswiler@toradex.com>
Date:   Tue Aug 14 00:35:56 2018 +0200

    ASoC: wm9712: fix replace codec to component
    
    commit 5e4cfadaf5b73a0801b2fa7fb007f98400ebfe6e upstream.
    
    Since commit 143b44845d87 ("ASoC: wm9712: replace codec to component")
    "wm9712-codec" got renamed to "wm9712-component", however, this change
    never got propagated down to the actual board/platform drivers. E.g. on
    Colibri T20 this lead to the following spew upon boot with sound/touch
    being broken:
    
    [    2.214121] tegra-snd-wm9712 sound: ASoC: CODEC DAI wm9712-hifi not registered
    [    2.222137] tegra-snd-wm9712 sound: snd_soc_register_card failed (-517)
    ...
    [    2.344384] tegra-snd-wm9712 sound: ASoC: CODEC DAI wm9712-hifi not registered
    [    2.351885] tegra-snd-wm9712 sound: snd_soc_register_card failed (-517)
    ...
    [    2.668339] tegra-snd-wm9712 sound: ASoC: CODEC DAI wm9712-hifi not registered
    [    2.675811] tegra-snd-wm9712 sound: snd_soc_register_card failed (-517)
    ...
    [    3.208408] tegra-snd-wm9712 sound: ASoC: CODEC DAI wm9712-hifi not registered
    [    3.216312] tegra-snd-wm9712 sound: snd_soc_register_card failed (-517)
    ...
    [    3.235397] tegra-snd-wm9712 sound: ASoC: CODEC DAI wm9712-hifi not registered
    [    3.248938] tegra-snd-wm9712 sound: snd_soc_register_card failed (-517)
    ...
    [   14.970443] ALSA device list:
    [   14.996628]   No soundcards found.
    
    This commit finally fixes this again.
    
    Signed-off-by: Marcel Ziswiler <marcel.ziswiler@toradex.com>
    Acked-by: Charles Keepax <ckeepax@opensource.cirrus.com>
    Signed-off-by: Mark Brown <broonie@kernel.org>
    Cc: stable@vger.kernel.org
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/sound/soc/codecs/wm9712.c b/sound/soc/codecs/wm9712.c
index 953d94d50586..ade34c26ad2f 100644
--- a/sound/soc/codecs/wm9712.c
+++ b/sound/soc/codecs/wm9712.c
@@ -719,7 +719,7 @@ static int wm9712_probe(struct platform_device *pdev)
 
 static struct platform_driver wm9712_component_driver = {
 	.driver = {
-		.name = "wm9712-component",
+		.name = "wm9712-codec",
 	},
 
 	.probe = wm9712_probe,
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1024_401e975e5950ad9f64f6348c21f6c05e65e0de91.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1024_401e975e5950ad9f64f6348c21f6c05e65e0de91.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
